L/Leptonica/build_tarballs.jl

```diff
@@ -46,12 +47,12 @@ dependencies = [
     Dependency(PackageSpec(name="JpegTurbo_jll", uuid="aacddb02-875f-59d6-b918-886e6ef4fbf8")),
     Dependency(PackageSpec(name="libpng_jll", uuid="b53b4c65-9356-5827-b1ea-8c7a1a84506f")),
     Dependency("Libtiff_jll"; compat="4.3.0"),
-    Dependency(PackageSpec(name="libwebp_jll", uuid="c5f90fcd-3b7e-5836-afba-fc50a0988cb2")),
+    Dependency("libwebp_jll"; compat="^1.2.4"),
```

Comment:
`^` is the default anyway 
```suggestion
    Dependency("libwebp_jll"; compat="1.2.4"),
```